components/common/EditNftModal.vue

```diff
@@ -16,10 +16,12 @@
           :label="$t('mint.nft.art.label')"
           required
         >
-          <FormLogoField
-            v-model:file="image"
-            v-model:url="imageUrl"
-          />
+          <NonRecommendFieldNotification :show="imageChanged">
+            <FormLogoField
+              v-model:file="image"
+              v-model:url="imageUrl"
+            />
+          </NonRecommendFieldNotification>
         </NeoField>
```

Comment:
I would show the message also in this state 

![CleanShot 2024-12-02 at 11 38 17@2x](https://github.com/user-attachments/assets/91a44af9-76b2-4240-a17c-b9a8cb52e65b)

Reply:
fixed